### ISSUE TITLE
Absent tear production on Schirmer I test

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11644]
**Preferred term label:**
Absent lacrimation on Schirmer I test

**Synonyms:**
* Absent tear flow on Schirmer I test
* 0 mm wetting on Schirmer I test
* Absolute aqueous tear deficiency on Schirmer I test

**Definition (free text, please give PubMed ID):**
The complete absence of aqueous tear production as clinically measured by a Schirmer I test result of 0 millimeters of wetting on the filter paper strip after 5 minutes. PMID: 39586848

**Parent term (use hpo.jax.org/app):**
Decreased lacrimation (HP:0000633) or Alacrima (HP:0000522)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
the HPO does not divide up phenotypes according to the mode of ascertainment (test). Instead I would suggest that we expand the definitions of the existing terms Decreased lacrimation (HP:0000633) or Alacrima (HP:0000522) accordingly and perhaps add this as a narrow synonym. Does that make sense?

**Comment by daboe01** [comment]
excellent idea! thank you

**Comment by pnrobinson** [comment]
updated definition and added synonyms.
